src/components/Common/seoLinks.js

```diff
@@ -21,19 +22,19 @@ export default function SeoLinks({ items, ...props }) {
     <PagePaths
       render={pathsById => (
         <List as="ul" {...props}>
-          {(items || []).map((item, i) => {
+          {(items || []).map(item => {
             const path = pathsById[item.id]
             if (path) {
```

Comment:
I'd replace by `{path && <ListItem key={generate()}>` for nicer refactoring reasons

Reply:
changed this for less repetition